matrix_sdk_base/src/rooms/normal.rs

```diff
@@ -533,4 +533,9 @@ impl RoomInfo {
 
         changed
     }
+
+    /// The number of active members (invited + joined) in the room.
+    pub fn active_members_count(&self) -> u64 {
+        self.summary.joined_member_count + self.summary.invited_member_count
```

Comment:
Technically this can overflow since both of those are u64, though they are casted from an `UInt` so realistically it can't. We should at least add a comment about this, perhaps we should also deal with it somehow if this ever becomes possible in the future.

Reply:
I see 3 methods that could be used:
- `u64::checked_add` returns an `Option`.
- `u64::wrapping_add` returns a wrapped value if `u64::MAX` is reached.
- `u64::overflowing_add` returns a tuple with the wrapped value and a boolean if the value overflowed.
- `u64::saturating_add` returns a value clamped between `0` and `u64::MAX`.

For me the first option seems like the most sensible. We could return a `Result` instead of the `Option` to make it clear that it's more of an error.

Reply:
What would you do with the error on its call site? I think a `saturating_add` is fine, we avoid the panic yet we return sensible values even in the unlikely event if we ever hit that condition.

Reply:
It was to let the user decide what to do in case of an error, but in our use case I would have probably saturated it too.

Reply:
Let's just saturate, rooms can't have that many members anyways and we don't want to let a server lying about the member count trigger a panic.